test/functional/sqlsrv/sqlsrv_fetch_object_unicode_col_name2.phpt

```diff
@@ -56,72 +57,119 @@ class Sample extends Product
     }
 }
 
+function getInputData1($inputs)
+{
+    return array('ID' => $inputs[0],
+                 'личное_имя'=> $inputs[1],
+                 'SafetyStockLevel' => $inputs[2],
+                 'StockedQty' => $inputs[3],
+                 'UnitPrice' => $inputs[4],
+                 'DueDate' => $inputs[5],
+                 'Color' => $inputs[6]);
+}
 
-require_once('MsCommon.inc');
-$tableName = "UnicodeColNameTest";
-
-include 'MsSetup.inc';
-
-$conn = connect(array( 'CharacterSet'=>'UTF-8' ));
-
-$tableName = "UnicodeColNameTest";
-
+function getInputData2($inputs)
+{
+    return array('SerialNumber' => $inputs[0],
+                 'Code'=> $inputs[1]);
+}
 
+require_once('MsCommon.inc');
+$conn = AE\connect(array('CharacterSet'=>'UTF-8'));
 
 // Create table Purchasing
 $tableName1 = "Purchasing";
 $tableName2 = "Country";
-dropTable($conn, $tableName1);
-dropTable($conn, $tableName2);
-$sql = "create table $tableName1 (ID CHAR(4), личное_имя VARCHAR(128), SafetyStockLevel SMALLINT,
-	StockedQty INT, UnitPrice FLOAT, DueDate datetime, Color VARCHAR(20))";
-sqlsrv_query($conn, $sql) ?: die(print_r(sqlsrv_errors(), true));
+
+$columns = array(new AE\ColumnMeta('CHAR(4)', 'ID'),
+                 new AE\ColumnMeta('VARCHAR(128)', 'личное_имя'),
+                 new AE\ColumnMeta('SMALLINT', 'SafetyStockLevel'),
+                 new AE\ColumnMeta('INT', 'StockedQty'),
+                 new AE\ColumnMeta('FLOAT', 'UnitPrice'),
+                 new AE\ColumnMeta('datetime', 'DueDate'),
+                 new AE\ColumnMeta('VARCHAR(20)', 'Color'));
+AE\createTable($conn, $tableName1, $columns);
 
 // Insert data
-$sql = "INSERT INTO $tableName1 VALUES
-	('P001','Pencil 2B','102','24','0.24','2016-02-01','Red'),
-	('P002','Notepad','102','12','3.87', '2016-02-21',Null),
-	('P001','Mirror 2\"','652','3','15.99', '2016-02-01',NULL),
-	('P003','USB connector','1652','31','9.99','2016-02-01',NULL)";
-sqlsrv_query($conn, $sql) ?: die(print_r(sqlsrv_errors(), true));
```

Comment:
what would happen if all insertions are done in one query?

Reply:
Is it possible for binding params?

Reply:
I haven't tried, but I would think something like this `$sql = 'INSERT INTO $tableName1 VALUES (?, ?), (?, ?)` should work.

Reply:
But that means I can't reuse the existing helper methods? Doing this or not doesn't change the essence of this particular test, right?

Reply:
I know it's a pain, but this seems to be the only test that does this (I haven't encounter another test that inserts multiple rows in one query). That's why I think it would be valuable to keep it.